src/components/Charts/ChartsTypes.tsx

```diff
@@ -0,0 +1,18 @@
+export enum ChartsType {
+    doubleSplineAreaChart = "Double Spline Area Chart"
+}
+
+ export enum ChartValueType {
+    Valuex = "ValueX",
```

Comment:
xAxisValue
yAxisValue
xAxisLabel
yAxisLabel

Don't add days in this enum

Reply:
got it.

Reply:
resolved .